twofactor_privacyidea/lib/Provider/TwoFactorPrivacyIDEAProvider.php

```diff
@@ -358,12 +377,13 @@ public function authenticate($username, $password) {
             $body = json_decode($body);
 
             if ($body->result->status === true) {
-                if ($pushResponse) {
-                    $error_message = $this->trans->t("The push token was not yet verified.");
+                if ($pushResponse === true || $tiqrResponse === true) {
+                    $error_message = $this->trans->t("Please confirm the authentication with your mobile device or enter a valid otp.");
```

Comment:
This message could be confusing, if I do not have an OTP token.

Reply:
Edited.